src/server/game/Battlegrounds/BattlegroundMgr.cpp

```diff
@@ -514,50 +543,10 @@ bool BattlegroundMgr::CreateBattleground(CreateBattlegroundData& data)
 {
     // Create the BG
     Battleground* bg = NULL;
```

Comment:
and here

Reply:
Still original code, if you have any suggestion to replace this it will be accepted.